### PR TITLE
feat: add convert to online data sample

### PR DESCRIPTION
**Pull request by foreverneverer**
see #46

### CODE REVIEW

File: pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/recipes/convertOnlineData/PegasusOnlineRDD.scala [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    this.metaServer = metaServer
+    this
+  }
+
+  def setTimeout(timeout: Long): OnlineConfig = {
+    this.timeout = timeout
+    this
+  }
+
+  def setAsyncWorks(asyncWorks: Int): OnlineConfig = {
```
Comment (neverchanje) [inline pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/recipes/convertOnlineData/PegasusOnlineRDD.scala]: AsyncWorkers

File: pegasus-spark-analyser/src/main/resources/log4j.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
-log4j.rootLogger=INFO,stdout
+log4j.rootLogger=INFO,stdout, rolling_file
```
Comment (neverchanje) [inline pegasus-spark-analyser/src/main/resources/log4j.properties]:
```suggestion
log4j.rootLogger=INFO,stdout,rolling_file
```

File: pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/PegasusWriter.scala [hunk 3]
```diff
@@ -0,0 +1,141 @@
+package com.xiaomi.infra.pegasus.spark.analyser
```
Comment (neverchanje) [inline pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/PegasusWriter.scala]: Put this file to pegasus-spark-common. It doesn't belong to analyser. Analyser should be responsible for snapshot analyzing only.

File: pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/PegasusWriter.scala [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  // default is 0.
+  var ttlThreshold: Int = 0
+  // flow control：
+  // single_qps = flowControl * bulkNum
+  // total_qps = single_qps * partitionNum(here partitionNum equal count of partition in
+  // running, also is count of parallel jobs, which isn't more than total pegasus partition count)
+  var bulkNum: Int = 10
+  var flowControl: Int = 5
+
+  def setMetaServer(metaServer: String): WriteConfig = {
```
Comment (neverchanje) [inline pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/PegasusWriter.scala]: scala auto-generates setter and getter by default for `var` type.

Comment (foreverneverer) [reply pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/PegasusWriter.scala]: `auto-generates` should not `return this`?

**Pull request closed without merging**